include/nihstro/inline_assembly.h

```diff
@@ -257,16 +257,20 @@ struct InlineAsm {
     }
 
     // Convenience constructors with implicit swizzle mask
+    // one source
```

Comment:
Having read this as "Conv ctors with implicit swizzle mask one source", I found it quite confusing. Can you remove the added comment, or come up with a clearer way to express this?

Reply:
I removed it.